### PR TITLE
styling

### PR DESCRIPTION
**Pull request by annakatarina1994**
Last minute styling for the homepage.

### CODE REVIEW

**Review by Nfada2010 — APPROVED** [review]

**Review by gaelanshively — APPROVED** [review]
super goods

File: frontend-spa/src/js/app.js [hunk 1]
```diff
@@ -20,6 +20,18 @@ const addEventListener = (query, functionToRun) => {
...
+
+const renderHome = () => {
+    clearView();
+const home = document.querySelector(".body");
+home.innerText= "The greatest Death Metal API of all time!";
+const homeImage = document.createElement("img");
+homeImage.setAttribute("src", "https://i.ya-webdesign.com/images/skull-horns-png-3.png");
+homeImage.setAttribute("alt", "Skull image");
+home.append(homeImage);
+homeImage.id = "homeImage";
```
Comment (ColinWake) [inline frontend-spa/src/js/app.js]:
```suggestion
homeImage.id = "homeImage";
home.append(homeImage);
```

**Review by ColinWake — CHANGES_REQUESTED** [review]

File: frontend-spa/src/js/app.js (hunk 1 above)
Comment (ColinWake) [reply frontend-spa/src/js/app.js]: Well done otherwise!

File: frontend-spa/src/js/app.js [hunk 2]
```diff
@@ -20,6 +20,18 @@ const addEventListener = (query, functionToRun) => {
     });
 };
 
+
+const renderHome = () => {
+    clearView();
+const home = document.querySelector(".body");
+home.innerText= "The greatest Death Metal API of all time!";
+const homeImage = document.createElement("img");
+homeImage.setAttribute("src", "https://i.ya-webdesign.com/images/skull-horns-png-3.png");
```
Comment (ColinWake) [inline frontend-spa/src/js/app.js]:
```suggestion
homeImage.setAttribute("src", "https://cdn.discordapp.com/attachments/689164265831399453/692857994211360778/repairedskull.png");
```

**Review by ColinWake — CHANGES_REQUESTED** [review]

**Review by ColinWake — APPROVED** [review]

**Review by caleb-bb — APPROVED** [review]

**Pull request merged**